test/algolia_test.exs

```diff
@@ -391,7 +397,7 @@ defmodule AlgoliaTest do
       |> batch_synonyms(synonyms, replace_existing_synonyms: true)
       |> wait()
 
-    hits = @settings_test_index |> export_synonyms() |> Enum.map(& &1)
+    hits = @settings_test_index |> export_synonyms(2) |> Enum.map(& &1)
```

Comment:
It isn't clear to me how this test tests the new code you added. It changes `hits_per_page`, but it doesn't seem to check the number of pages or the number of hits returned.

Reply:
Good point, I updated the test to be more explicit